src/hooks/useLoadDocuments.ts

```diff
@@ -18,14 +18,18 @@ export interface DocumentFromServer {
 }
 
 export const formatServerResponse = (documents: DocumentFromServer[]): Document[] =>
-  documents.map(document => ({
-    ...document,
-    article: ARTICLES[document.article],
-    alternatives: document.alternatives.map(it => ({
-      article: ARTICLES[it.article],
-      word: it.alt_word,
-    })),
-  }))
+  documents
+    .filter(it => ARTICLES[it.article])
+    .map(document => ({
+      ...document,
+      article: ARTICLES[document.article],
+      alternatives: document.alternatives
+        .filter(it => ARTICLES[it.article])
```

Comment:
We should not filter here, as this will remove words with pluarl article for all exercise, they should only be removed from the ArticleChoiceExercise, for Wordlist/WordChoiceExercise and WriteExercise they should still be there and tested.